doc/specs/stdlib_math.md

```diff
@@ -275,3 +275,75 @@ program demo_logspace_rstart_cbase
 
 end program demo_logspace_rstart_cbase
 ```
+## `arange`
+
+### Status
+
+Experimental
+
+### Class
+
+Pure function.
+
+### Description
+
+Create a vector of the `integer/real` type with given fixed spaced values within a given interval.
```

Comment:
```suggestion
Creates an array of the `integer/real` type with fixed-spaced values of given spacing, within a given interval.
```

Reply:
In fact, what `arange` returns is a one-dimensional `vector`, the word `array` may be ambiguous?

Reply:
To me it's the other way around--I think of vector as a mathematical concept and array as a programming one. I think array is also a more Fortrannic word. But it's not a big deal, just a suggestion, pick your favorite, either works well 👍

Reply:
I think it can be taken as `one-dimensional array`